cmd/security-spire-config/seed_builtin_entries.sh

```diff
@@ -34,8 +34,6 @@ for dockerservice in security-spiffe-token-provider support-notifications suppor
     # Temporary workaround because service name in dockerfile is not consistent with service key.
     # TAF scripts depend on legacy docker-compose service name. Fix in EdgeX 3.0.
     service=`echo -n ${dockerservice} | sed -e 's/app-service-/app-/'`
```

Comment:
Also can remove this tweak not that all services App Services use `app-`

Reply:
I think this is beyond scope of this issue related PR: if we have application services that are not started with `app-service-` in docker-compose service names then it is a different issue we are dealing with here for support services.

Reply:
You can easily change the scope of this PR to be `remove unnecessary service name tweaks` so to reduce the need to do another PR.

Reply:
@lenny-intel   this PR is for issue #4507